CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 Version master (UNRELEASED)
 ---------------------------
 
-- Adds disk usage retreival methods to user and workflow models.
+- Adds disk usage retrieval methods to user and workflow models.
+- Changes human-readable size units to display them in powers of 1024.
```

Comment:
Since there were no quota units in 0.7.0, there is no change when compared to the previous release... So instead of Changes, it should say Adds.  But you can simply join with the previous item, perhaps something like:

- Adds new disk usage retrieval methods using canonical (bytes) and human-readable (KiB) units. (``User``, ``Workflow``)